modules/volunteers/admin/sitesupport/giftTable.js

```diff
@@ -260,6 +244,57 @@ export default {
         return [u.length, nonGroup];
       }
       return [0, 1];
+    },
+    sortTable(column) {
+      if (column.unsortable) {
+        return;
+      }
+      if (column.value === this.sortColumn) {
+        this.decendingSort = !this.decendingSort;
+      }
+      this.sortColumn = column.value;
+      let targetLeft = (this.decendingSort) ? 1 : -1;
+      let targetRight  = (this.decendingSort) ? -1 : 1;
```

Comment:
I think we might be able to simplify this a little bit for ascending/descending sort. Let me know what you think.

First, it seems like we're trying to determine whether we're sorting ascending or descending order.

We might be able to use something like `const sortDirection = this.descendingSort ? -1 : 1;`

Then, we can take the values of our return statements and multiply them by the sort direction to get ascending or descending order.

Line 272 could become: `return (parseFloat(a.value) - parseFloat(b.value)) * sortDirection;`
Line 274 could become `return a.name.localeCompare(b.name) * sortDirection;`
Line 276 could become `return (aValue - bValue) * sortDirection;`

So as an example, let's imagine we're doing ascending order sort - I will just cover line 272.

a.value = 10
b.value = 5

(10 - 5) * 1 = 5, so `a` should be sorted lower than `b`.

c.value = 5
d.value = 7

(5 - 7) * 1 = -2, so `c` should be sorted higher than `d`.

If we flip the `sortDirection` value to `-1` we should get the opposite ordering.